examples/todo-intermediate/src/controllers/todo-list.controller.ts

```diff
@@ -0,0 +1,67 @@
+import {repository, HasManyRepository} from '@loopback/repository';
+import {TodoListRepository} from '../repositories';
+import {TodoList, Todo} from '../models';
+import {requestBody, param, post, get, del, put, patch} from '@loopback/rest';
+
+export class TodoListController {
+  protected todoRepo: HasManyRepository<TodoList>;
+
+  constructor(
+    @repository(TodoListRepository) protected todoListRepo: TodoListRepository,
+  ) {}
+
+  @post('/todo-lists')
+  async createTodoList() {
+    const newTodoList = {createdAt: new Date()};
+    return await this.todoListRepo.create(newTodoList);
+  }
+
+  @get('/todo-lists/{id}')
+  async findTodoListById(@param.path.number('id') id: number) {
+    return await this.todoListRepo.findById(id);
+  }
+
+  @get('/todo-lists')
+  async findTodos() {
```

Comment:
`findTodoLists`?

TBH, I don't see much value in repeating the model name in every controller method - the model name is already captured in the class name. The current version of the REST Controller template used by `lb4 controller` seems to follow this convention too.

Could you please create the core of this new controller by calling `lb4 controller` instead of modifying code copied from `TodoController`?

Reply:
In that case, I think it makes sense to also modify Controller methods in `Todo` to be built from `lb4 controller (CRUD)` command. Thoughts?

Reply:
> In that case, I think it makes sense to also modify Controller methods in `Todo` to be built from `lb4 controller (CRUD)` command, maybe in a separate PR. Thoughts?

That would be great 👍